src/promptflow-devkit/promptflow/_cli/_pf/_service.py

```diff
@@ -284,10 +304,11 @@ def show_service():
         log_file = HOME_PROMPT_FLOW_DIR / PF_SERVICE_LOG_FILE
     else:
         log_file = get_current_env_pfs_file(PF_SERVICE_LOG_FILE)
+    extra_info = {"log_file": log_file.as_posix(), "local_promptflow_version": get_pfs_version()}
```

Comment:
add promptflow version here, user can call /heartbeat to see pfs version.
Actually, if  promptflow version is not the same with pfs versio, we will show warning message when start trace